### ISSUE TITLE
Cancel button for cancelling dashboard filter edits not working 

### ISSUE THREAD

**Issue by TLazarevic** [metabase/metabase#27424]
**Describe the bug**
After editing a dashboard filter and trying to cancel the changes, 'Cancel' button in sidebar for filter edits doesn't seem to be working.

**Logs**
Console log caused by a button click:

```
Uncaught TypeError: e.props.setParameter is not a function
    at ParameterSidebar.jsx:43:16
    at Object.c (react-dom.production.min.js:14:84)
    at f (react-dom.production.min.js:14:238)
    at react-dom.production.min.js:14:292
    at M (react-dom.production.min.js:15:72)
    at ct (react-dom.production.min.js:52:170)
    at at (react-dom.production.min.js:51:255)
    at lt (react-dom.production.min.js:52:334)
    at bt (react-dom.production.min.js:56:10)
    at N (react-dom.production.min.js:287:189)
```

**To Reproduce**
1. Navigate to a dashboard with filters
2. Click on 'Edit dashboard'
3. Click on a dashboard filter
4. Make some changes (e.g. update dashboard filter Label in sidebar)
5. Click on 'Cancel' button in the sidebar
6. Nothing happens

**Expected behavior**
Sidebar is closed and changes to the filter are reverted

**Information about your Metabase Installation:**

Browser: Brave v1.46.144
Metabase version: v0.45.0

**Severity**
Not severe since there is an option for cancelling all dashboard edits.

**Comment by paoliniluis** [comment]
Hi, I can't reproduce this on 45.1. Can you check if this happens on 45.1?

**Comment by flamber** [comment]
Fixed by #27170, as the cancel button doesn't have any real function, so it was removed.
